_data/jobs.yml

```
@@ -1,3 +1,7 @@
+- expires: 2021-08-10
+  location: San Diego, CA or remote in US
+  name: Bioinformatics Staff Scientist (Ph.D.) or Software Engineer: API interoperability
```

You just need some quotes!

```suggestion
  name: "Bioinformatics Staff Scientist (Ph.D.) or Software Engineer: API interoperability"
```